integration_tests/run_tests.py

```diff
@@ -9,15 +9,19 @@
     "expr_03.py",
     "expr_04.py",
     "expr_05.py",
+    "modules_01.py",
+    #"modules_02.py",
 ]
 
 def main():
     print("Compiling...")
     for pyfile in tests:
         basename = os.path.splitext(pyfile)[0]
-        cmd = os.path.join("src", "bin", "lpython") + " integration_tests/%s -o integration_tests/%s" % (pyfile, basename)
+        cmd = os.path.join("..", "src", "bin", "lpython") + " %s -o %s" % (pyfile, basename)
         print("+ " + cmd)
+        os.chdir("integration_tests")
         r = os.system(cmd)
+        os.chdir("..")
```

Comment:
This part is needed because the module import is looking for the module in the current directory instead of the directory relative to the file that contains the "import ..." statement. We have to fix that too in subsequent PRs.